native/ex_tokenizers/src/tokenizer.rs

```diff
@@ -551,10 +554,14 @@ pub fn tokenizer_decode_batch(
         }
     });
 
-    Ok(tokenizer
-        .resource
-        .0
-        .decode_batch(sentences, opts.skip_special_tokens)?)
+    Ok(tokenizer.resource.0.decode_batch(
+        sentences
+            .iter()
+            .map(Vec::as_slice)
+            .collect::<Vec<&[u32]>>()
+            .as_slice(),
+        opts.skip_special_tokens,
+    )?)
```

Comment:
Pretty sure this is the only reasonable way to get from a vec of vecs to a slice of slices. If anybody has any other ideas I'm all ears.